tests/unit/models/primo.js

```diff
@@ -23,7 +23,18 @@ describe("Primo Model >", function() {
         "coverImageUrl": "https://assets.thirdiron.com/default-journal-cover.png",
         "browzineEnabled": false,
         "externalLink": "http://za2uf4ps7f.search.serialssolutions.com/?V=1.0&N=100&L=za2uf4ps7f&S=I_M&C=0096-6762"
-      }]
+        },
+        {
+          "id": 13016,
+          "type": "journals",
+          "title": "International Journal of Cancer",
+          "issn": "00207136",
+          "sjrValue": 2.259,
+          "coverImageUrl": "https://assets.thirdiron.com/images/covers/0020-7136.png",
+          "browzineEnabled": true,
+          "browzineWebLink": "https://develop.browzine.com/libraries/XXXX/journals/13016?utm_source=api_716"
```

Comment:
Not sure I fully follow why we added a new journal here.  Could we have just added a new article that is associated with one of the other journals, but that has an expressionOfConcernNoticeUrl ?